Source/Scene/UrlTemplateImageryProvider.js

```diff
@@ -568,6 +578,16 @@ define([
             }
             that._credit = credit;
 
+            if (properties.customTags) {
+                for (var tag in properties.customTags) {
+                    if (properties.customTags.hasOwnProperty(tag)) {
+                        var targetTag = '{' + tag + '}';
+                        tags[targetTag] = properties.customTags[tag]; //eslint-disable-line no-use-before-define
```

Comment:
I'm surprised these eslint directives are needed.  What does it thing is being used before define?

Reply:
Here was the messages:
```
/home/travis/build/AnalyticalGraphicsInc/cesium/Source/Scene/UrlTemplateImageryProvider.js
  585:25  error  'tags' was used before it was defined              no-use-before-define
  586:25  error  'pickFeaturesTags' was used before it was defined  no-use-before-define
```

Reply:
Let's remove the directives and simple move `tags` and `pickFeaturesTags` to the top of the file.